apiserver/facades/client/action/operation_test.go

```diff
@@ -199,9 +199,6 @@ func (s *enqueueSuite) setupMocks(c *gc.C) *gomock.Controller {
 	s.Authorizer.EXPECT().HasPermission(gomock.Any(), gomock.Any(), gomock.Any()).Return(nil).AnyTimes()
 	s.Authorizer.EXPECT().AuthClient().Return(true)
 
-	s.model = action.NewMockModel(ctrl)
-	s.model.EXPECT().ModelTag().Return(s.modelTag).MinTimes(1)
```

Comment:
You will probably have to replace this with a mock of the model service for the same information I suspect? I dare say the tests are going to fail without this.

Reply:
We removed model tag from model, and in the `SetUpSuite`, the model tag has already been assigned to the suite. So it makes sense and tests go well.